ironfish-cli/src/commands/deposit-all.ts

```diff
@@ -202,6 +202,11 @@ export default class DepositAll extends IronfishCommand {
       unconfirmedBalance = CurrencyUtils.decode(balanceResp.content.unconfirmed)
       pendingBalance = CurrencyUtils.decode(balanceResp.content.pending)
 
+      if (confirmedBalance < 0 || unconfirmedBalance < 0 || pendingBalance < 0) {
+        await PromiseUtils.sleep(30000)
+        continue
+      }
```

Comment:
It seems unusual for any balances to be negative -- Does it fix itself eventually if it continues like this? Otherwise, I wonder if it's a bug in the wallet, and if it's better to exit and tell users to rescan their accounts.